src/applications/login/components/SignInWrapper.jsx

```diff
@@ -1,9 +1,5 @@
 import React from 'react';
 
 export default function SignInWrapper({ children }) {
-  return (
```

Comment:
This is removed from the login application entry, as this markup is now included in the `LoginContainer`